sklearnex/basic_statistics/basic_statistics.py

```diff
@@ -14,4 +14,130 @@
 # limitations under the License.
 # ==============================================================================
 
-from onedal.basic_statistics import BasicStatistics
+import numpy as np
+from sklearn.base import BaseEstimator
+from sklearn.utils import check_array, deprecated
```

Comment:
please use onedal.utils._validation _check_array instead
https://github.com/intel/scikit-learn-intelex/blob/01df2bb15360481441736882d9847d9adf24c811/onedal/utils/validation.py#L125

Reply:
currently check_array is used much more often than _check_array in sklearnex folder. why should it be changed here? probably the best way is to do follow-up PR with standardization of the usage across the whole repo.